polygon/sync/store.go

```diff
@@ -52,13 +52,18 @@ type bridgeStore interface {
 	ReplayInitialBlock(ctx context.Context, block *types.Block) error
 }
 
-func NewStore(logger log.Logger, executionStore executionStore, bridgeStore bridgeStore) *ExecutionClientStore {
+type wiggleCalculator interface {
```

Comment:
Seems overkill to me - this is the only usage. We can just use the struct.

Reply:
I prefer using interfaces to keep the code testable.